.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'monthly'
+    groups:
+      github-actions:
+        patterns:
+          - '*'
```

Comment:
I think I would still weakly prefer to add this config section:
- If we ever do pin an action to a minor or patch version, it means we won't suddenly see an increase in dependabot PRs as a result
- It's more explicit

I don't feel strongly, though. I'm mainly suggesting this because it's what we have at CPython, and that seems to work pretty well for us over there :)

Reply:
Cool, I'm on board. Thanks for your quick reviews and feedback!